subsys/fs/shell.c

```diff
@@ -505,7 +505,7 @@ static int cmd_mount_fat(const struct shell *shell, size_t argc, char **argv)
 	res = fs_mount(&fatfs_mnt);
 	if (res != 0) {
 		shell_error(shell,
-			"Error mounting fat fs.Error Code [%d]", res);
```

Comment:
If you are touching this make `fat` `FAT`

Reply:
Done!

Reply:
Does not seem to change.